database/migrations/2022_02_28_170001_create_subnets_v6_table.php

```diff
@@ -22,6 +23,11 @@ public function up()
                 ->constrained('jeoip_ip2location_locations')
                 ->onUpdate('cascade')
                 ->onDelete('cascade');
+            $table->foreignIdFor(Asn::class, 'asn_id')
```

Comment:
Jeoip is deployed and running, we can't edit migrations that already ran on production.

Reply:
i removed that and create new migration file for add ASN column